terminus-ui/select/src/select.component.ts

```diff
@@ -2000,6 +2006,7 @@ export class TsSelectComponent implements
 
     // Mark this control as 'touched' to trigger any validations needed on blur
     this.onTouched();
+    this.updateValueAndValidity();
```

Comment:
Not sure about best practice, but would it be worthwhile to call this.updateValueAndValidity() within this.onTouched() method?

Reply:
We actually cannot do this. 😞  The `onTouched` method is something we have to define as part of the CVA implementation. Angular's core forms code will call our `registerOnTouched` method which will overwrite our `onTouched` method with theirs.